label_sync/main.go

```diff
@@ -846,17 +873,10 @@ func getTextColor(backgroundColor string) (string, error) {
 }
 
 func writeCSS(tmplPath string, outPath string, config Configuration) error {
-	var prLabels []Label
-	prLabels = append(prLabels, LabelsForTarget(config.Default.Labels, bothTarget)...)
-	prLabels = append(prLabels, LabelsForTarget(config.Default.Labels, prTarget)...)
-
-	for repo := range config.Repos {
-		prLabels = append(prLabels, LabelsForTarget(config.Repos[repo].Labels, bothTarget)...)
-		prLabels = append(prLabels, LabelsForTarget(config.Repos[repo].Labels, prTarget)...)
```

Comment:
The new func does not filter out labels for the issue target like the old code does. Is that intended?

Reply:
Sorry, forgot to call that out... it wasn't clear to me why we were intentionally filtering out issue labels? Avoiding unused css classes on the PR page? Didn't seem worth the complexity